.github/actions/install-dependencies/action.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Install dependencies'
+description: 'Install dependencies to build shadow-utils'
+runs:
+  using: "composite"
+  steps:
+  - shell: bash
+    run: |
+      sudo apt-get update -y
+      sudo apt-get install -y automake autopoint byacc expect gettext \
+      libselinux1-dev libtool xsltproc
```

Comment:
After reviewing [PR524](https://github.com/shadow-maint/shadow/pull/524) I wonder if I should execute `apt-get -y ubuntu-dev-tools && apt-get -y build-dep shadow` instead of this command.

Reply:
(thought i had replied to this)

That woudl be fine.  I had been keeping deps installation to a minimum when I was trying to run the tests in containers on my laptop (while on limited internet), but in CI it's probably better to just do a build-dep install.